.github/ISSUE_TEMPLATE/pkg-request.yml

```
@@ -0,0 +1,67 @@
+name: Package Request
+description: Request the addition of a package
+labels: ["request"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        ### Don't request an update of a package,
+        [We have a script for that](https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt).
+        However, a quality pull request may help.
+  - id: name
+    type: input
+    attributes:
+      label: Package name
+      placeholder: foobar9k
+  - id: description
+    type: textarea
+    attributes:
+      label: Description
+      description: What does the package do?
+      placeholder: >
+        Foobar9k is a music player that turns your music up to 11.
+        It provides features X, Y, and Z, which other music players in Void don't.
+  - type: markdown
+    attributes:
+      value: >
+        ## Quality Requirements
+
+        To be included in the Void repository, software must meet **at least one** of the following requirements.
+        Exceptions to the list are possible, and might be accepted, but are extremely unlikely.
+        If you believe you have an exception, start a PR and make an argument for why that particular piece of software,
+        while not meeting any of the following requirements, is a good candidate for the Void packages system.
```

```suggestion
        while not meeting any of the following requirements, is a good candidate for inclusion in Void packages.
```
This sounds better to me, but I am not a native speaker, what do you think?

---

Your suggestion sounds better to me